/right_rs_pointcloud_to_laserscan:
  ros__parameters:
    angle_increment: 0.00872664625
    angle_max: -0.785
    angle_min: -2.355
    inf_epsilon: 1.0
    max_height: 1.5
    min_height: 0.0
    queue_size: 11
    range_max: 5.0
    range_min: 0.0
    scan_time: 0.03333333333333333
    target_frame: 'base_footprint' # Transforming to target frame base_link puts scan in correct frame. Keeping it in original frame gets really messed up. Not sure why.
    transform_tolerance: 1.0
    use_inf: true
    use_sim_time: true
